tests/sitemaps_tests/test_http.py

```diff
@@ -259,19 +259,95 @@ def test_empty_sitemap(self):
 
     @override_settings(LANGUAGES=(('en', 'English'), ('pt', 'Portuguese')))
     def test_simple_i18nsitemap_index(self):
-        "A simple i18n sitemap index can be rendered"
+        """
+        A simple i18n sitemap index can be rendered.
+        """
```

Comment:
Should there be a test that the variables are as expected (`i18n` `alternates` etc)?

Reply:
Thanks for all the feedback! I'm not sure I understand this one; I followed the existing tests as examples.

The fact, that `i18n` and `alternates` are enabled can be deducted from the returned XML. Do you mean to validate the values of the attributes on eg. the `tests.sitemaps_tests.urls.http.AlternatesI18nSitemap` in the test fixture?

The `django.contrib.sitemaps.views.sitemap` view does not pass the `Sitemap` instance with those attributes to the template context. Is there another way to validate those from a response object?

Reply:
Thanks for picking these up so quickly. :-)

I was trying to pick up on Simon's earlier comment. Maybe it's not needed or possible. 

I'd suggest unticking 'needs improvement' on trac now to get this back in the review queue?

Reply:
> I'd suggest unticking 'needs improvement' on trac now to get this back in the review queue?

i did. let's get this done :)